docker/dbt/jaffle_shop/models/marts/core/dim_order_dates.sql

```diff
@@ -1,5 +1,5 @@
 {{
-    config(materialized = 'incremental')
+    config(materialized = 'incremental', schema='custom_schema')
```

Comment:
please do create a different model for showcasing this functionality

Reply:
Will do.